src/pybind/mgr/dashboard/controllers/smb.py

```diff
@@ -184,3 +184,18 @@ def delete(self, cluster_id: str, share_id: str):
         resource['share_id'] = share_id
         resource['intent'] = Intent.REMOVED
         return mgr.remote('smb', 'apply_resources', json.dumps(resource)).one().to_simplified()
+
+
+@UIRouter('/smb')
+class SMBStatus(RESTController):
+    @EndpointDoc("Get SMB feature status")
+    @Endpoint()
+    @ReadPermission
+    def status(self):
+        status = {'available': False, 'message': None}
+        try:
+            mgr.remote('smb', 'show', ['ceph.smb.cluster'])
+            status['available'] = True
+        except (ImportError, RuntimeError):
+            status['message'] = 'Module not found'
```

Comment:
This looks like a vague message. Either we can be more descriptive here or we don't even need the message because the header you put below `SMB manager module is not enabled` itself is more clear and the showing the `Enable` button is pretty much self explanatory of what the action is.

```suggestion
            status['message'] = 'Please enable the SMB module for SMB Management.'
```